Dockerfile

```diff
@@ -1,5 +1,22 @@
-FROM cypress/base:10
+FROM node:11.6
```

Comment:
Should we bump the node version used in deployments then as well? ( https://github.com/binxhealth/frontend/blob/master/Dockerfile  specifies node:10.12.0-alpine )

Reply:
Yea, I will when I create a PR on frontend to test this.